src/site/markdown/server/manage-webadmin.md

```diff
@@ -2515,3 +2516,34 @@ Response codes:
 
  - 200: A list of corresponding tasks is returned
  - 400: Invalid status value
+
+## Cassandra extra operations
+
+Some webadmin features to manage some extra operations on Cassandra tables, like solving inconsistencies on projection tables.
```

Comment:
what are the causes of such inconsistencies ?
what about protecting against several concurrent tasks ?